base/exports.jl

```diff
@@ -397,6 +397,8 @@ export
     reim,
     reinterpret,
     rem,
+    rem1,
+    rem2pi,
```

Comment:
Honestly, I'm not sure. I'm also not sure why `rem2pi` takes a rounding argument but `mod2pi` doesn't.